packages/taikoon-ui/src/components/sections/Collapsible.section.svelte

```diff
@@ -0,0 +1,87 @@
+<script lang="ts">
+  import { slide } from 'svelte/transition';
+
+  import { Icons } from '$components/core/Icons';
+  import { classNames } from '$lib/util/classNames';
+  import { Section } from '$ui/Section';
+  import { H1 } from '$ui/Text';
+  export let options: {
+    visible: boolean;
+    title: string;
+    content: string;
+  }[] = [];
+
+  const PlusIcon = Icons.PlusSign;
+  const XIcon = Icons.XSolid;
+</script>
+
+<Section height="fit">
+  <div
+    class={classNames(
```

Comment:
Dont use "inline" CSS, export them to variables.

Also the bridge uses daisyUI, which you also have as dependency. There is a collapse component included.